frontend/src/components/ShareTweet.vue

```diff
@@ -25,16 +25,36 @@ const isShareable = computed(() => {
 });
 
 const tweetText = computed(() => {
-  const sourceToken = transfer.value?.sourceAmount.token;
-  const sourceChain = transfer.value?.sourceChain;
-  const targetChain = transfer.value?.targetChain;
-  const transferTime = transfer.value?.transferTimeSeconds;
+  if (!transfer.value) {
+    return '';
+  }
+  const sourceToken = transfer.value.sourceAmount.token;
+  const sourceChain = transfer.value.sourceChain;
+  const targetChain = transfer.value.targetChain;
+  const transferTime = transfer.value.transferTimeSeconds;
 
-  return `I just used @beamerbridge to seamlessly and securely transfer #${sourceToken?.symbol} from ${sourceChain?.name} to ${targetChain?.name} in ${transferTime} seconds! 🔥
+  const defaultText = `I just used @beamerbridge to seamlessly and securely transfer #${sourceToken.symbol} from ${sourceChain.name} to ${targetChain.name} in ${transferTime} seconds! 🔥
 
-Unlock lightning-fast and secure bridging with Beamer today 💪💫 and don't miss out on Beamer's #layerupfest! #beam2L2
+Unlock lightning-fast and secure bridging with Beamer today 💪💫  Now also live on Polygon zkEVM!
 https://app.beamerbridge.com/
 `;
+
+  const subsidizedTransferText = `Insane! @beamerbridge unlocked 🦓  - we don't know for how long, so don't sleep on it 👀
+
+I just securely transferred #${sourceToken.symbol} from ${sourceChain.name} to ${targetChain.name} in ${transferTime} seconds with 🦓 and you can do too! 🔥
+
+Unlock lightning-fast and secure bridging with Beamer today 💪💫  
+https://app.beamerbridge.com/
+`;
+
+  let text: string;
+  if (isSubsidizedTransfer(transfer.value) && transfer.value.fees.uint256.isZero()) {
+    text = subsidizedTransferText;
+  } else {
+    text = defaultText;
```

Comment:
Nitpick: We can simply write `return defaultText;` and `return subsidizedTransferText;` here.